server/test/article.spec.js

```diff
@@ -173,156 +173,173 @@ describe('Articles routes', () => {
       expect(res).to.have.status(422);
       expect(message).to.be.equal('title, body, description and tagList are required');
     });
+  });
+
+  describe('GET /api/v1/article/:page', () => {
     it('it should get all articles', async () => {
       const res = await chai.request(app)
-        .get('/api/v1/article')
-        .set({ Authorization: `Bearer ${userToken}` });
+        .get('/api/v1/article/1');
       const { message } = res.body;
       expect(res).to.have.status(200);
       expect(message).to.be.equal('Articles successfully retrieved');
       expect(res.body.data).to.have.property('articles');
```

Comment:
Good job @BukkyOmo. I think we can make this test a bit more comprehensive by checking against actual articles retrieved. You could check against `title` for example from the seeded data.

Reply:
Fixed. Thanks